scripts/run_eval.py

```
@@ -0,0 +1,12 @@
+from dreem.training import train
```

Ruff

`dreem.training.train` imported but unused

Remove unused import: `dreem.training.train`

(F401)